cleaners/windows_defender.xml

```diff
@@ -12,20 +12,55 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+    @app Microsoft Windows Defender
+    @url https://www.microsoft.com/en-us/safety/pc-security/windows-defender.aspx
+    @os Windows
+    @cleanerversion v1.5.0
+    @cleanerdate 2019-03-13
+    @cleanerby ??? & https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-13
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
 
 -->
 <cleaner id="windows_defender" os="windows">
   <label>Windows Defender</label>
   <description>Anti-virus</description>
   <option id="history">
     <label>History</label>
-    <description>Delete the files</description>
+    <description>Delete the history</description>
     <action command="delete" search="walk.files" path="%COMMONAPPDATA%\Microsoft\Windows Defender\Scans\History\Results\Quick"/>
     <action command="delete" search="walk.files" path="%COMMONAPPDATA%\Microsoft\Windows Defender\Scans\History\Results\Resource"/>
   </option>
+  <option id="temp">
+    <label>Temporary files</label>
+    <description>Delete the temporary files</description>
+    <action command="delete" search="file" path="%windir%\Temp\MpCmdRun.log"/>
+    <action command="delete" search="file" path="%windir%\Temp\MpSigStub.log"/>
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-d.exe"/>
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-fe.exe"/>
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-fe_bd.exe"/>
+    <!-- AS Engine: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine.exe"/>
+    <!-- AS Engine Patch: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine_Patch_*.exe"/>
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base.exe"/>
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base_Patch*.exe"/>
```

Comment:
When using a wildcard, use `search="glob"`

Reply:
Did it in the last version...